pkg/send/sender.go

```diff
@@ -44,7 +45,6 @@ type Payload struct {
 	AddEdges    []tr.Edge `json:"addEdges,omitempty"`    // List of Edges which must be added
 	DeleteEdges []tr.Edge `json:"deleteEdges,omitempty"` // List of Edges which must be deleted
 	ClearAll    bool      `json:"clearAll,omitempty"`    // Tells the aggregator to clear existing data first.
```

Comment:
So `ClearAll` is still included in the payload. **That's actually good** because it reduces the risk while we update the code in the indexer. Let's try to come back an clean-up after the indexer updates.